<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LongitudeInputClass</class>
 <widget class="QWidget" name="LongitudeInputClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">LongitudeInput</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSpinBox" name="degreesInput">
     <property name="toolTip">
      <string>&lt;html&gt;Degrees&lt;/html&gt;</string>
     </property>
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="minutesInput">
     <property name="toolTip">
      <string>&lt;html&gt;Arc minutes (1/60 degree)&lt;/html&gt;</string>
     </property>
     <property name="suffix">
      <string>'</string>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="secondsInput">
     <property name="toolTip">
      <string>&lt;html&gt;Arc seconds (1/60 arc minute or 1/3600 degree)&lt;/html&gt;</string>
     </property>
     <property name="suffix">
      <string>&quot;</string>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="signInput">
     <item>
      <property name="text">
       <string>East</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>West</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
